homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -42,10 +43,8 @@ async def async_setup_entry(
     """Set up the sensor from a config entry created in the integrations UI."""
     coordinator = hass.data[DOMAIN][config_entry.entry_id]
 
```

Comment:
```suggestion

    unique_id = config_entry.unique_id
    if TYPE_CHECKING:
        assert unique_id
```

Reply:
Does this use `assert` outside unit testing? This is new syntax for me ;)

Reply:
yep haha